plugins/WebsiteType/lang/en.json

```diff
@@ -0,0 +1,7 @@
+{
+  "WebsiteType": {
+    "Website": "Website",
+    "Websites": "Websites",
+    "WebsiteDescription": "A website or a mobile web app that is usually consumed via a browser"
```

Comment:
@mattab you might want to have a look here re translation. Also further below in `plugin.json`